tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -1341,6 +1341,7 @@ public function dataFileAsserts(): iterable
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/nullsafe-vs-scalar.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-8517.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/../Rules/Functions/data/bug-9803.php');
+		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-9939.php');
```

Comment:
This needs to be put in `PHP_VERSION_ID` if.

Reply:
Also it's better to put in some random place in the middle, because at the end it's more likely to have conflict when merging into 1.11.x